coderd/provisionerdserver/provisionerdserver_test.go

```diff
@@ -855,3 +814,10 @@ func setup(t *testing.T, ignoreLogErrors bool) *provisionerdserver.Server {
 		Auditor:      mockAuditor(),
 	}
 }
+
+func must[T any](value T, err error) T {
+	if err != nil {
+		panic(err)
```

Comment:
If you look at where I do it, it is used for `json.Marshal` on a standard struct. This should absolutely never fail. I cannot add another arg, because you cannot do:

```golang
must(t, json.Marshal())
```

You can't use multiple returns and an argument like that :cry:. The `must` is a nice way to keep the context in the struct body.